pkg/scheduler/plugins/predicates/proportional.go

```diff
@@ -33,13 +33,13 @@ func checkNodeResourceIsProportional(task *api.TaskInfo, node *api.NodeInfo, pro
 			return status, nil
 		}
 	}
+
 	for resourceName, resourceRate := range proportional {
 		if value, found := node.Idle.ScalarResources[resourceName]; found {
 			cpuReserved := value * resourceRate.CPU
 			memoryReserved := value * resourceRate.Memory * 1000 * 1000
-			r := node.Idle.Clone()
-			r = r.Sub(task.Resreq)
-			if r.MilliCPU < cpuReserved || r.Memory < memoryReserved {
+
+			if node.Idle.MilliCPU-task.Resreq.MilliCPU < cpuReserved || node.Idle.Memory-task.Resreq.Memory < memoryReserved {
```

Comment:
@Cdayz Hi, I'm sorry for the late review. Can you demonstrate how this introduce panic? I've not gotten the point here.

Reply:
if node.idle has some dimension less than requst, there will be a assert.

Reply:
> if node.idle has some dimension less than requst, there will be a assert.

OK. That's reasonable. I'm OK about the fix.